QML_HUD/main.cpp

```
@@ -7,36 +7,115 @@
 #include <QQmlApplicationEngine>
 #include <QQmlContext> 
 #include <QObject>
+#include <QFileSystemWatcher>
 #include <QDebug>
+#include <QVariantMap>
+#include <QWindow>
 
-// TODO: Umiescic w QMLach wyswietlacz wartosci i jednostki
-// TODO: Mirror
+// X TODO: Umiescic w QMLach wyswietlacz wartosci i jednostki
 // TODO: Poprawic rysowanie niektorych kontrolerow
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_ENUM is a macro then please configure it.

(unknownMacro)